### ISSUE TITLE
Быстрый выбор цвета для фигур.

### ISSUE THREAD

**Issue by Fen8fen** [SBProX/SBProX-Dev#855]
Здравствуйте уважаемые разработчики.
Предложение добавить 2 ячейки быстрого выбора цвета для фигур. (Также как и с линиями)
Из за того что выбор цвета, для линий происходит быстро. Буквально в 2 клика, с платформой комфортно работать.
Думаю достаточно будет 2 или 3 ячейки.

![image](https://user-images.githubusercontent.com/20509322/68837279-325f0c00-06f7-11ea-94a9-9f1e14c7e247.png)

**Comment by alex12mosk** [comment]
Плюсую. Лучше сделать ячеек 6. А еще лучше выкинуть бесполезные наборы палитр и вместо них сделать кастомную палитру, чтоб туда можно было сохранить свои цвета.

**Comment by jekksnn** [comment]
Согласен, огромные палитры и большое их количество лучше заменить (дополнить) палитрой в которую можно сохранить несколько своих цветов. И еще сама технология выбора цвета несколько громоздка а именно: открываешь свойства выбираешь палитру нажимаешь применить (при этом фигура уже окрашивается в цвет) а потом еще нажимаешь "Ок" Зачем последняя операция? 
А вот с линиями как раз обратная проблема: выбираешь свойства и если нажал выбор цвета, то окно закрывается без "Ок", а ведь там и толщина линии и текст и другие настройки (но эти настройки нужно делать до выбора цвета иначе будешь открывать заново). Нельзя ли сделать выбор настроек линии и потом "Ок" означает применение всех настроек?

**Comment by Fen8fen** [comment]
jekksnn said: _А вот с линиями как раз обратная проблема:_ 

с линиями нет проблем. 90 % открытие меню это выбор цвета.
Пользуйтесь первой главной ячейкой выбора цвета, там и толщину линии поменяете.

![image](https://user-images.githubusercontent.com/20509322/69233178-6b9bee00-0bc7-11ea-9c48-bbd0a873274d.png)
